src/dda/cli/inv/__init__.py

```diff
@@ -40,14 +40,11 @@ def cmd(app: Application, *, args: tuple[str, ...], no_dynamic_deps: bool) -> No
             features.append("legacy-test-infra-definitions")
         elif task.startswith("system-probe."):
             features.append("legacy-btf-gen")
-        elif task.startswith("kmt."):
-            features.append("legacy-kernel-matrix-testing")
 
     if no_dynamic_deps:
         import sys
 
         app.subprocess.replace_current_process([sys.executable, "-m", "invoke", *args])
-        return
```

Comment:
My editor spotted that this is unreachable code based on the type annotations but I'm not sure why CI doesn't catch that, perhaps that has to be explicitly configured. I will investigate at a later time.